# Study 899 — Cash + Call "90/10" 📈

**Bill Gross's "90/10": park ~90% in T-bills so your capital comes back to par, and spend ~10%
"renting" convex upside with call options. If the market crashes the calls expire worthless and the
bills carry you; if it rockets you get leveraged, capped-loss upside. Does "protect capital, rent
upside" actually beat just holding stocks on a risk-adjusted basis?**

Free listed-option history doesn't exist, so the ~10% convex sleeve is a **documented proxy**: a
rolling **1-year at-the-money SPY call, marked daily with Black–Scholes** (strike = spot at each
annual roll, priced off SPY's trailing realized vol and the ^IRX bill rate; the 10% budget buys as
much notional as the fair price affords, capped near 100% of capital). Real tape SPY/BIL,
2007-05-31 → 2026-06-30.

*Numbers below are the frozen headline (`docs/results.md`, fingerprint `b954e845292f`); the live cells
run the fast synthetic control. Short history: BIL lists 2007 — a single, GFC-anchored cycle (which
is exactly where a capital floor should shine). The Black–Scholes mark **flatters** the strategy
(realized-vol pricing, no dividend drag) — the premium sweep restores reality.*


In [1]:
R = {'alpha_ann': 0.46,
 'asym': -0.003,
 'avg_w': 0.68,
 'bear_prot': 0.369,
 'bear_prot_min': 0.043,
 'bear_w': 0.41,
 'beta': 0.638,
 'bh_cagr': 10.7,
 'bh_dd': -55.2,
 'bh_sharpe': 0.543,
 'bh_sortino': 0.513,
 'bh_vol': 19.78,
 'boot_hi': 0.21,
 'boot_lo': -0.334,
 'boot_point': -0.051,
 'boot_win': 33.8,
 'calm_w': 0.79,
 'crash08_bh': -36.8,
 'crash08_bh_dd': -47.1,
 'crash08_c': -8.1,
 'crash08_c_dd': -10.2,
 'crash20_bh': 18.3,
 'crash20_bh_dd': -33.7,
 'crash20_c': -1.1,
 'crash20_c_dd': -15.7,
 'crash22_bh': -18.2,
 'crash22_bh_dd': -24.5,
 'crash22_c': -17.1,
 'crash22_c_dd': -19.6,
 'diff_t_nw': -1.37,
 'dn_cap': 0.568,
 'end': '2026-06-30',
 'era_e_bh': 0.335,
 'era_e_n': 2163,
 'era_e_t': 0.94,
 'era_e_tt': 0.453,
 'era_e_vs': 0.119,
 'era_l_bh': 0.767,
 'era_l_n': 2635,
 'era_l_t': 1.68,
 'era_l_tt': 0.999,
 'era_l_vs': 0.232,
 'fp': 'b954e845292f',
 'n_days': 4799,
 'n_rolls': 19,
 'n_seeds': 30,
 'pm080_sh': 0.629,
 'pm080_vs': 0.087,
 'pm100_sh': 0.492,
 'pm100_vs': -0.051,
 'pm125_sh': 0.335,
 'pm125_vs': -0.208,
 'pm150_sh': 0.193,
 'pm150_vs': -0.35,
 'pm200_sh': -0.039,
 'pm200_vs': -0.582,
 'sharpe_vs_bh': -0.051,
 'start': '2007-05-31',
 'static_cagr': 8.09,
 'static_dd': -40.2,
 'static_sharpe': 0.543,
 'static_vol': 13.44,
 't_alpha': 0.33,
 'tt_cagr': 6.13,
 'tt_dd': -19.8,
 'tt_sharpe': 0.492,
 'tt_sortino': 0.497,
 'tt_vol': 10.42,
 'turnover': 0.18,
 'up_cap': 0.564}

## 1. Capital protection really works — that part is true

The profile does exactly what it says on the tin: it floors the drawdown. Over 2007–2026 the 90/10 book's worst peak-to-trough was **a third** of buy-and-hold's, at **half the volatility** — the calls simply expired worthless in the crashes while the bills carried the book.

In [2]:
print(f"90/10  : maxDD {R['tt_dd']:.1f}%   vol {R['tt_vol']:.1f}%")
print(f"S&P 500: maxDD {R['bh_dd']:.1f}%   vol {R['bh_vol']:.1f}%")
print(f"2008 crash: S&P {R['crash08_bh']:+.1f}% (DD {R['crash08_bh_dd']:.1f}%)"
      f"  ->  90/10 {R['crash08_c']:+.1f}% (DD {R['crash08_c_dd']:.1f}%)")

90/10  : maxDD -19.8%   vol 10.4%
S&P 500: maxDD -55.2%   vol 19.8%
2008 crash: S&P -36.8% (DD -47.1%)  ->  90/10 -8.1% (DD -10.2%)


## 2. …but it doesn't actually *beat* stocks risk-adjusted — it ties

The pitch is a better *risk-adjusted* outcome. It isn't there. Once you measure both books in **excess of cash**, 90/10's **Sharpe 0.49** essentially **ties** buy-and-hold's **0.54** — the gap is a rounding error, and a bootstrap can't tell them apart (P(90/10 wins) ~34%). You gave up **4.6%/yr of compounding** (6.1% vs 10.7%) to reshape the *tail*, not to earn more per unit of risk.

In [3]:
print(f"excess Sharpe : 90/10 {R['tt_sharpe']:.3f}   vs   S&P 500 {R['bh_sharpe']:.3f}   (gap {R['sharpe_vs_bh']:+.3f})")
print(f"CAGR         : 90/10 {R['tt_cagr']:.2f}%   vs   S&P 500 {R['bh_cagr']:.2f}%")
print(f"bootstrap 95% CI on the Sharpe gap: [{R['boot_lo']:+.3f}, {R['boot_hi']:+.3f}]  "
      f"P(90/10 wins) {R['boot_win']:.1f}%  -> a statistical tie")

excess Sharpe : 90/10 0.492   vs   S&P 500 0.543   (gap -0.051)
CAGR         : 90/10 6.13%   vs   S&P 500 10.70%
bootstrap 95% CI on the Sharpe gap: [-0.334, +0.210]  P(90/10 wins) 33.8%  -> a statistical tie


## 3. And a *real* option costs more than our model assumed

Our call is priced off **realized** volatility. A real listed call trades at **implied** vol, which is systematically *higher* (the variance risk premium — that's how option sellers get paid). Charge the option what it actually costs — a **1.25–1.5×** markup — and the tie flips to a clear **loss** vs buy-and-hold. Renting upside is a negative-carry trade priced to leave the buyer behind.

In [4]:
for lbl,sh,vs in [('BS-fair (realized vol)',R['pm100_sh'],R['pm100_vs']),
                  ('1.25x (mild VRP)',R['pm125_sh'],R['pm125_vs']),
                  ('1.50x (typical)',R['pm150_sh'],R['pm150_vs'])]:
    print(f"  option cost {lbl:24s}: 90/10 Sharpe {sh:+.3f}   vs S&P {vs:+.3f}")

  option cost BS-fair (realized vol)  : 90/10 Sharpe +0.492   vs S&P -0.051
  option cost 1.25x (mild VRP)        : 90/10 Sharpe +0.335   vs S&P -0.208
  option cost 1.50x (typical)         : 90/10 Sharpe +0.193   vs S&P -0.350


## 4. A live synthetic control — the engine is honest

We plant a **bear** world (capital must be protected) and a **calm** steady bull (nothing to protect, so the premium just bleeds), and check the machinery reads each correctly. No network.

In [5]:
import os, sys
sys.path.insert(0, os.path.abspath('..'))
sys.path.insert(0, os.path.abspath(os.path.join('..','..','..')))
from cash_call import data, strategy as st
bear = st.synthetic_detect(data.synthetic_prices(seed=899, n_days=2500, drift=-0.0004, sigma=0.016)[0])
calm = st.synthetic_detect(data.synthetic_prices(seed=899, n_days=2500, drift=0.0005, sigma=0.006)[0])
print(f"bear world : drawdown protection {bear['dd_protection']:+.3f}, equity de-risked to {bear['avg_weight']:.2f}  (capital floored)")
print(f"calm bull  : 90/10 Sharpe {calm['sharpe_tt']:+.3f} vs S&P {calm['sharpe_bh']:+.3f}, equity weight {calm['avg_weight']:.2f}  (premium bleeds, upside given up)")

bear world : drawdown protection +0.465, equity de-risked to 0.30  (capital floored)
calm bull  : 90/10 Sharpe +0.422 vs S&P +0.364, equity weight 0.68  (premium bleeds, upside given up)


## 5. The honest verdict

- **Signal: Weak.** The capital protection is **real and mechanical** (maxDD -19.8% vs -55.2%, 2008 cut from −47% to −10%). But the *risk-adjusted-return* claim isn't there: excess Sharpe **0.49 vs 0.54** is a statistical tie, the option's convexity adds no alpha (*t* = +0.33), and that's at a premium that *flatters* the strategy.
- **Tradability: Mirage.** The parity survives only at the fictional fair price — a realistic option markup (1.25–1.5×) sinks it clearly below buy-and-hold. It's not a trading-cost story (one roll a year); it's that renting upside is negative-carry. You run 90/10 to **sleep at night**, not to get paid.